### PR TITLE
Add PosAngle model and a few small changes

### PR DESCRIPTION
**Pull request by cquiroz**
Move these from explore and ags

### CODE REVIEW

File: modules/core/shared/src/main/scala/lucuma/core/enum/CloudExtinction.scala [hunk 1]
```diff
@@ -6,9 +6,9 @@ package lucuma.core.enum
 import lucuma.core.util.Display
 import lucuma.core.util.Enumerated
 
-sealed abstract class CloudExtinction(val toDeciMagnitudes: Int) extends Product with Serializable {
-  def toMagnitudes: Double = toDeciMagnitudes / 10.0
-  def label: String        = f"< $toMagnitudes%.1f mag"
+sealed abstract class CloudExtinction(val toDeciBrightness: Int) extends Product with Serializable {
```
Comment (cquiroz) [inline modules/core/shared/src/main/scala/lucuma/core/enum/CloudExtinction.scala:9]: This change is to make the naming consistent

**Review by swalker2m — APPROVED** [review]
Looks good to me. :+1:

File: modules/core/shared/src/main/scala/lucuma/core/enum/GuideProbe.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+object GuideProbe {
+
+  /** @group Constructors */
+  case object AOWFS extends GuideProbe("AOWFS")
+
+  /** @group Constructors */
+  case object OIWFS extends GuideProbe("OIWFS")
+
+  /** @group Constructors */
+  case object PWFS extends GuideProbe("PWFS")
```
[the remark this answers is not in the document]
Comment (cquiroz) [reply modules/core/shared/src/main/scala/lucuma/core/enum/GuideProbe.scala:24]: Right, it is coming from the ocs and I was starting to use it in AGS, but maybe we need to review it, I'll remove  it

File: modules/core/shared/src/main/scala/lucuma/core/model/PosAngle.scala [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+import monocle.Focus
+import monocle.Lens
+import monocle.Optional
+import monocle.Prism
+import monocle.macros.GenPrism
+
+/**
+  * Position Angle model
+  */
+sealed trait PosAngle
```
[the remark this answers is not in the document]
Comment (cquiroz) [reply modules/core/shared/src/main/scala/lucuma/core/model/PosAngle.scala:18]: I guess we should add it

**Pull request merged**